content/realtime/channel-lifecycle.textile

```diff
@@ -0,0 +1,141 @@
+
+The "REST API":/rest-api can also be used "enumerate all active channels":/rest/channel-lifecycle#enumeration-rest, or obtain the status of an "individual channel":#metadata-rest. At present, REST "requests":/rest/usage#request for lifecycle events will not count towards your message limit. If you are making use of "Reactor":/reactor however, then each message sent over Reactor *will* count towards your message limits.
+
+h2(#use-cases). Use cases
+
+Having access to channel metadata can provide numerous benefits. In a scenario where the number of subscribers to a channel goes well beyond a hundred, usage of other options such as "presence":/core-features/presence become less effective. You could instead make use of our "occupancy events":#occupancy and "channel metadata":#metachannels to check the number of active subscribers, getting accurate values easily.
+
+Equally, you may want to only publish data when you know someone is subscribed for your data. Therefore, "channel lifecycle events":#lifecycle-events can notify you when a channel becomes active, and can notify you when the last subscriber leaves the channel.
+
+If you need to be able to query channel metadata at any point, you can make use of the "channel metadata REST APIs":/rest/channel-lifecycle#metadata-rest to inspect the state of individual channels, or "enumerate":/rest/channel-lifecycle#enumeration-rest all active channels in the system.
+
+h2(#tutorials). Tutorials
+
+Check out some of our "tutorials":/tutorials to see examples of using metachannels and channel lifecycles.
+
+Through a realtime connection you can subscribe to both "channel lifecycle":#lifecycle-events and "occupancy":#occupancy events. This is done through a channel's "metachannel":#metachannels.
+
+h2(#metachannels). Metachannels
+
+Metachannels are a namespace of channels which all start with the @[meta]@ qualifier, uniquely identifying them from regular channels. An example of a metachannel would be @[meta]channel.lifecycle@.
+
+There are a number of metachannels available, which are:
+
+- log := This metachannel is used to broadcast log messages (usually error messages) for events that occur within the application's context
+- channel.lifecycle := This metachannel carries messages about channel lifecycle and metadata
+- connection.lifecycle := This metachannel carries messages about the lifecycle of realtime connections
+
+h2(#lifecycle-events). Lifecycle events
+
+The following events arise as messages on the channel lifecycle metachannel. In all cases, the @data@ member of the message is a "ChannelDetails":#channel-details instance.
```

Comment:
Perhaps instead of saying 'on the channel lifecycle metachannel' - give the actual channel name embedded in @@?